store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -125,4 +101,14 @@ public String call() throws Exception {
         return sanitize();
     }
 
+    protected Set<String> getSkipTagsUnclosedCdata() {
+        Set<String> zimbraSkipTagsWithUnclosedCdata = new HashSet<>();
+        try {
+            zimbraSkipTagsWithUnclosedCdata = new HashSet<>(
+                    Arrays.asList(LC.zimbra_skip_tags_with_unclosed_cdata.value().toString().split(",")));
+        } catch (NullPointerException ex) {
```

Comment:
Strange to see these days developers at Zimbra are catching `Exception` and when the reviewer asked to catch a specific exception `NullPointerException` was the best choice by the developer which satisfied the reviewer too! What a lovely days!